### ISSUE TITLE
error when I load my data

### ISSUE THREAD

**Issue by liamxg** [abschneider/StrainHub#9]
Dear @colbyford, and @abschneider,

Could you please help me out:

<img width="1020" alt="image" src="https://github.com/user-attachments/assets/6ae91d68-4ba7-4008-ac12-5f79657c4845">

Best,
Liam

**Comment by colbyford** [comment]
Yes, you need to check that all of the Accessions (the taxa names on the tips of your tree) match the Accessions in your metadata file. (Check for things like hyphens, underscores, or periods that would make the Accessions not match.

**Comment by liamxg** [comment]
Dear @colbyford,

please see below:

<img width="877" alt="image" src="https://github.com/user-attachments/assets/b7bcf430-1728-44fe-b570-4e295dcf4296">


Best,
Liam

**Comment by liamxg** [comment]
Dear @colbyford,

Is there any command line version for this, because my data is quite big, thanks.

<img width="513" alt="image" src="https://github.com/user-attachments/assets/cb7dde24-36af-45da-aa1e-96cd2e1bb5d7">


Best,
Liam
